tests/build.go

```diff
@@ -376,3 +379,39 @@ func testBuildLabelNoKey(t *testing.T, sb integration.Sandbox) {
 	require.Error(t, err, string(out))
 	require.Equal(t, strings.TrimSpace(string(out)), `ERROR: invalid key-value pair "=TEST_STRING": empty key`)
 }
+
+func testBuildCacheExportNotSupported(t *testing.T, sb integration.Sandbox) {
+	if sb.Name() != "docker" {
+		t.Skip("skipping test for non-docker workers")
+	}
+
+	dir := createTestProject(t)
+	cmd := buildxCmd(sb, withArgs("build", "--cache-to=type=registry", dir))
+	out, err := cmd.CombinedOutput()
+	require.Error(t, err, string(out))
+	require.Contains(t, string(out), "Cache export is not supported")
+}
```

Comment:
Wondering if we should start using a test-table for these tests; did a quick try what it'd look like; https://github.com/dvdksn/buildx/pull/2

WDYT @crazy-max ?

Reply:
Yes sounds good but I think we should split ones about empty key from other tests

Reply:
Opened as https://github.com/docker/buildx/pull/2041

> but I think we should split ones about empty key from other tests

Any specific reason for that? (just wondering)